### ISSUE TITLE
Produce table output

### ISSUE THREAD

**Issue by sectorsize512** [sectorsize512/mentorship#9]
Currently it is hard for a user to quickly get a sense of how good is the list. Producing the following table will help:

```
Mentee     | Mentor       | In-person-virtual match | Industry-Academia Match | Not same institution | Topic Match |
John Smith | Angela Davis |        x                |           x             |       x              |    x        |
Peter Cool | Camilla Uga  |        x                |           x             |       x              |     -       |
...
```

**Comment by tarver52** [comment]
Сheck if it's done !

**Comment by sectorsize512** [comment]
Why is the name of the output file hardcoded - "rez_table.csv"?

**Comment by sectorsize512** [comment]
Fixed in 905653eba820a88d9834c0f487ece550e4defc7f
